pydicom/overlays/numpy_handler.py

```diff
@@ -43,47 +37,25 @@
 except ImportError:
     HAVE_NP = False
 
-import pydicom.uid
 from pydicom.pixel_data_handlers.numpy_handler import unpack_bits
 
+if TYPE_CHECKING:
+    from pydicom.dataset import Dataset
```

Comment:
It really bugs me that I don't know how to test this line, even nominally. Any advice?

Maybe I can monkey patch `typing`...

Reply:
I'm not sure which line you mean -- testing the `TYPE_CHECKING` path?

Reply:
Soemthing like this:
```py
from importlib import reload
import typing
from pydicom.overlays import numpy_handler
    
def test_type_checking(monkeypatch):
    assert not hasattr(numpy_handler, 'Dataset')
    monkeypatch.setattr(typing, 'TYPE_CHECKING', True)
    reload(numpy_handler)
    assert hasattr(numpy_handler, 'Dataset')
```

Reply:
Ah. neat! Thanks